examples/classifying_shapes.ipynb

```diff
@@ -59,7 +59,8 @@
    "outputs": [],
```

Comment:
<p>Would it be simpler to say "we will only consider the homology dimensions 0 (connected components), 1 (loops), and 2 (voids)."?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I'm trying not to use "voids" for 2D as I need a word for the generic dimension and I chose to use "voids". If you agree that this is reasonable, I could use "cavities" instead for the 2D case?</p>

Reply:
<p>Sorry, I hadn't realised this was not the new notebook. I agree with your suggestion then.</p>

Reply:
<p>Done.</p>